### PR TITLE
Added instruction to obtain module name

### PR DESCRIPTION
**Pull request by fruit-ninja**
It was unclear in the documentation on how to locate the module name.

## Purpose of this pull request

This pull request (PR) ...

## Affected DevDocs pages

-  ...

## Links to Magento source code

-  ...

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by rogyar** [comment]
Example from the codebase 

https://github.com/magento/magento2/blob/245d4efc4c0ccd0b0fe1de4c099634d01c742869/setup/src/Magento/Setup/Console/Command/ModuleStatusCommand.php#L47

This command will show all the modules as two groups:
- Modules enabled
- Modules disabled

File: src/extensions/install/index.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ Module is disabled
 ```
 
 {:.bs-callout-info}
-The extension name is in the format `<VendorName>_<ComponentName>`; it's not the same format as the Composer name. Use this format to enable the extension.
+The extension name is in the format `<VendorName>_<ComponentName>`; it's not the same format as the Composer name. Use this format to enable the extension. If you are not sure what the extension name is, you can run `bin/magento module:status` and look for the extension under List of disabled modules.
```
Comment (rogyar) [inline src/extensions/install/index.md:114]:
```suggestion
The extension name is in the format `<VendorName>_<ComponentName>`; it's not the same format as the Composer name. Use this format to enable the extension. If you are not sure what the extension name is, you can run `bin/magento module:status` and look for the extension under "List of disabled modules".
```

**Comment by dobooth** [comment]
running tests

**Comment by dobooth** [comment]
running tests

**Pull request merged**